doc/neps/nep-template.rst

```diff
@@ -13,15 +13,20 @@ Abstract
 
 The abstract should be a short description of what the NEP will achieve.
 
-
-Detailed description
---------------------
+Motivation
+----------
 
 This section describes the need for the NEP.  It should describe the existing
-problem that it is trying to solve and why this NEP makes the situation better.
-It should include examples of how the new functionality would be used and
-perhaps some use cases.
+problem, who it affects, what it is trying to solve, and why.
+It should include examples of how the new functionality would be used, intended
+use-cases and psuedo-code describing its use.
 
+Literature Review
+-----------------
+
+This section should list relevant and/or similar technologies, possibly in other
+libraries. It does not need to be comprehensive, just list the major examples of
+prior and relevant art.
 
```

Comment:
I think "Detailed description" should be kept here. And pseudo-code / examples should stay here.